Project.toml

```
@@ -10,7 +10,7 @@ SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 
 [compat]
 ExpectationStubs = "0.2.0"
-HTTP = "0.7.0, 0.8"
+HTTP = ">= 0.8.5"
```

```suggestion
HTTP = "0.8.5"
```

Wr stick to semver bounds as I don't like my code just breaking.
And I don't like having to retroactively edit the registry to stop it downgrading to the last version without bounds when it does break